include/sensor/pmw3360.h

```diff
@@ -14,16 +15,16 @@
 
 #include <zephyr/drivers/sensor.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 /**
  * @defgroup pmw3360 PMW3360 motion sensor driver
  * @{
  * @brief PMW3360 motion sensor driver.
  */
 
+#ifdef __cplusplus
```

Comment:
Please align the approach for this file with paw3212 header.

Reply:
updated